tests/storage/migration.go

```diff
@@ -78,8 +78,7 @@ var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSched
 			WorkloadUpdateMethods: []virtv1.WorkloadUpdateMethod{virtv1.WorkloadUpdateMethodLiveMigrate},
 		}
 		rolloutStrategy := pointer.P(virtv1.VMRolloutStrategyLiveUpdate)
-		config.PatchWorkloadUpdateMethodAndRolloutStrategy(originalKv.Name, virtClient, updateStrategy, rolloutStrategy,
-			[]string{virtconfig.VolumesUpdateStrategy, virtconfig.VolumeMigration})
+		config.PatchWorkloadUpdateMethodAndRolloutStrategy(originalKv.Name, virtClient, updateStrategy, rolloutStrategy, []string{})
```

Comment:
Can't we make the tests parallel now?

Reply:
We are still patching kubevirt CR for the rolloutStrategy. But yes, I will investigate this in a separate PR